docs/_docs/04_community/contributing.md

```diff
@@ -150,16 +149,60 @@ to include the following:
    test output so we can verify that everything is working.
 1. Any notes on backwards incompatibility or downtime.
 
+#### Validate the Pull Request for Azure Platform
+
+If you're contributing code for the [Azure Platform](https://azure.com) and if you have and active _Azure subscription_, it's recommended to follow the below guidelines after [creating a pull request](https://help.github.com/articles/creating-a-pull-request/). If you're contributing code for any other platform (e.g., AWS, GCP, etc), you can skip these steps.
+
+> Once the PR has `Azure` tag and _Approved_, following pipeline will run automatically on an active Azure subscription, which [Microsoft](https://microsoft.com) provides.

```

Comment:
Prob worth calling out that Terratest maintainers will add this tag to automatically run tests for the PR, but if you want to run the tests yourself before then, you can follow the steps below.

Reply:
I just updated, thanks for the feedback